### PR TITLE
feat: clean snapshot is added automatically

### PR DESCRIPTION
**Pull request by ireznice**

### CODE REVIEW

File: lib/online_kitchen/labmanager4.rb [hunk 1]
```diff
@@ -75,6 +75,25 @@ def equip_machine_start(name)
...
+        req1.content_type = 'application/json'
+        req1.body = '{"name":"clean"}'
+        req1.basic_auth config[:username], config[:password]
+        request_id = Net::HTTP.start(uri.hostname, uri.port, config.http_opts.symbolize_keys) do |http|
+          response = http.request(req1)
+          JSON.parse(response.body)['responses'][0]['request_id']
+        end
+
+        OnlineKitchen.logger.debug("waiting for clean snapshot to be created")
+        wait_for_machine_released(config, request_id)
```
Comment (petrSchreiber) [inline lib/online_kitchen/labmanager4.rb:93]: I would propose to rename the method `wait_for_machine_released` to `wait_for_request_completion`.

File: lib/online_kitchen/workers/lab_manager_equip.rb [hunk 2]
```diff
@@ -69,7 +70,15 @@ def machine_processable?(machine)
     def perform_start(machine, equip_arg)
       OnlineKitchen::LabManager4.equip_machine_start(machine.provider_id)
       self.class.perform_in(
-        rand(3..10).seconds,
+        rand(2..7).seconds,
```
Comment (petrSchreiber) [inline lib/online_kitchen/workers/lab_manager_equip.rb:73]: Nice speedup, thanks!

**Review by petrSchreiber — CHANGES_REQUESTED** [review]
Minor request to clarify function purpose

File: lib/online_kitchen/workers/lab_manager_equip.rb [hunk 3]
```diff
@@ -69,7 +70,15 @@ def machine_processable?(machine)
...
+        equip_arg.merge('started' => true)
+      )
+    end
+
+    def perform_create_clean_snapshot(machine, equip_arg)
+      OnlineKitchen::LabManager4.equip_create_clean_snapshot(machine.provider_id)
+      self.class.perform_in(
+        rand(2..5).seconds,
         equip_arg.merge('started' => true)
```
[the remark this answers is not in the document]
Comment (ireznice) [reply lib/online_kitchen/workers/lab_manager_equip.rb:82]: this is one of the things you should have detected :)

**Review by petrSchreiber — APPROVED** [review]
Tested change, thank you very much. It will increase the usability immensely.

**Pull request merged**